### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by oroboro-cpu**

### CODE REVIEW

File: src/main/java/core/basesyntax/Figures.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    }
+
+    public String getColor() {
+        return color.toString();
+    }
+
+    public abstract double getArea();
+
+    @Override
+    public void draw() {
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/Figures.java:17]: remove this method, we are not obligated to override interfaces method in the abstract class if we gonna do it in child classes

File: src/main/java/core/basesyntax/Figures.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    public Figures(Color color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color.toString();
+    }
+
+    public abstract double getArea();
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/Figures.java:14]: create a separate interface and move this method there

**Review by RomanDubovskyi — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Drawable.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    public void draw();
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/Drawable.java:4]: methods in interface are public abstract by default

File: src/main/java/core/basesyntax/Figures.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figures implements Drawable, Displayable, Calculable {
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/Figures.java:3]:
```suggestion
public abstract class Figure implements Drawable, Displayable, Calculable {
```

**Review by RomanDubovskyi — APPROVED** [review]

File: src/main/java/core/basesyntax/Drawable.java (hunk 3 above)
Comment (oroboro-cpu) [reply src/main/java/core/basesyntax/Drawable.java]: All corrections are done. Thank you.

**Pull request closed without merging**